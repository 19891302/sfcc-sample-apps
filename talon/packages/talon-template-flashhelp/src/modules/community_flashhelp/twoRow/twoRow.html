<template>
    <slot name="title"></slot>
    <slot name="main"></slot>
</template>