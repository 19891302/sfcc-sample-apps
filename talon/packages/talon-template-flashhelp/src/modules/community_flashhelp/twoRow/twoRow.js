import { LightningElement } from 'lwc';

export default class TwoRow extends LightningElement {}