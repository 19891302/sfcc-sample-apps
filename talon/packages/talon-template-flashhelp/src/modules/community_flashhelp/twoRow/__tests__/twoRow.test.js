import { createElement } from 'lwc';
import twoRow from 'community_flashhelp/twoRow';

describe('community_flashhelp/twoRow', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-two-row', { is: twoRow });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});