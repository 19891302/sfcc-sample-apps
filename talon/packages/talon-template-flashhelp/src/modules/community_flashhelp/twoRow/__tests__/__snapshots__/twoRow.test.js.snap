// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/twoRow renders correctly 1`] = `
<community_flashhelp-two-row>
  #shadow-root(open)
    <slot
      name="title"
    />
    <slot
      name="main"
    />
</community_flashhelp-two-row>
`;
